tests/unit_tests/lcls_tools/superconducting/test_sc_cryomodule.py

```diff
@@ -6,13 +6,13 @@
 
 class TestCryomodule(TestCase):
     def test_is_harmonic_linearizer_true(self):
-        hl = MACHINE.cryomodules[f"H{randint(1,2)}"]
+        hl = MACHINE.cryomodules[f"H{randint(1, 2)}"]
         self.assertTrue(
             hl.is_harmonic_linearizer, msg=f"{hl} is_harmonic_linearizer is not true"
         )
 
     def test_is_harmonic_linearizer_false(self):
-        cm = MACHINE.cryomodules[f"{randint(1, 35):02d}"]
+        cm = MACHINE.cryomodules[f"{randint(1, 35): 02d}"]
```

Comment:
There shouldn't be a space between the colon and the number here. This changes the format string from
`01` to ` 01`. The dictionary `MACHINE.cryomodules` doesn't expect a leading space in its key names.

Reply:
ok I addressed it, lets see if the tests pass